src/MarginTrading.Backend.Core/OrderExtensions.cs

```diff
@@ -159,6 +158,7 @@ public static void UpdateClosePrice(this IOrder order, decimal closePrice)
             if (orderInstance != null)
             {
                 orderInstance.ClosePrice = closePrice;
+                orderInstance.FplData.ActualHash++;
```

Comment:
Why is field named Hash, when it's not a hash actually?

Reply:
Because it is named so in account fpl data. Was copy pasted )